mmdet/models/losses/gfocal_loss.py

```diff
@@ -184,6 +188,24 @@ def forward(self,
                 calculate_loss_func = quality_focal_loss_with_prob
             else:
                 calculate_loss_func = quality_focal_loss
+
+            if isinstance(target, torch.Tensor):
+                # this means that target is already in One-Hot form.
```

Comment:
QFL uses soft targets instead of binary One-Hot labels. Please fix the comments.

Reply:
Yes